src/plugins/auto/executable_network.cpp

```diff
@@ -265,6 +267,24 @@ MultiDeviceExecutableNetwork::MultiDeviceExecutableNetwork(const std::string&
         _inferPipelineTasksDeviceSpecific["CPU_HELP"] = nullptr;
         _executor->run(_loadContext[CPU].task);
         _executor->run(_loadContext[ACTUALDEVICE].task);
+        // create task for resource recyle
+        auto recycleTask = [this]() mutable {
+            size_t destroynum = 0;
+            std::unique_lock<std::mutex> lock(_recycleMutex);
+            while (!_exitFlag && !_loadContext[ACTUALDEVICE].isAlready)
+                _recycleCond.wait(lock);
+            while (!_exitFlag) {
+                // clean up helper heap
+                WorkerInferRequest *workerRequestPtr = nullptr;
+                if (_idleWorkerRequests[_loadContext[CPU].workName].try_pop(workerRequestPtr))
```

Comment:
still need to sleep at least 1ms while try pop failed

Reply:
I need to work/test on this part and sleep thing carefully, looks like it will affect perf data with small iterations.